### ISSUE TITLE
Seldon operator is missing from Openshift 4.9 catalog

### ISSUE THREAD

**Issue by samuelvl** [SeldonIO/seldon-core#3751]
Seldon operator is missing from the certified operators catalog in Openshift 4.9:

```shell
$ oc get packagemanifest seldon-operator-certified                              
Error from server (NotFound): packagemanifests.packages.operators.coreos.com "seldon-operator-certified" not found
```

I am running this command from a Openshift 4.9.5 cluster:

```shell
$ oc version                                       
Client Version: 4.9.5
Server Version: 4.9.5
Kubernetes Version: v1.22.0-rc.0+a44d0f0
```

When will be there a compatible release available?

**Comment by cliveseldon** [comment]
We will need to look further into this after the next release 1.12in which all CRDs and webhook should be compatible. The certification process has changed so we need to do some work before again we can be certified.

**Comment by cliveseldon** [comment]
No ready to be merged on  [community](https://github.com/k8s-operatorhub/community-operators/pull/608) and [community-prod](https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/647). Certified being worked on.

**Comment by cliveseldon** [comment]
No merged as a [certified operator for Openshift 4.9](https://github.com/redhat-openshift-ecosystem/certified-operators/pull/405)
